tests/release_candidate_scraper/test_release_candidate_scraper.py

```diff
@@ -7,128 +7,67 @@
-     'url': 'firefox/candidates/23.0.1-candidates/build3/mac/en-US/Firefox 23.0.1.dmg'},
-    # -a firefox -p win32 -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1-build3.en-US.win32.exe',
-     'url': 'firefox/candidates/23.0.1-candidates/build3/win32/en-US/Firefox Setup 23.0.1.exe'},
-    # -p win64 -v 23.0.1
-    {'args': {'platform': 'win64',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1-build3.en-US.win64.exe',
-     'url': 'firefox/candidates/23.0.1-candidates/build3/win64/en-US/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win32 -v 23.0.1 -l de
-    {'args': {'application': 'firefox',
-              'locale': 'de',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1-build3.de.win32.exe',
-     'url': 'firefox/candidates/23.0.1-candidates/build3/win32/de/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win32 -v 23.0.1 --build-number=1
-    {'args': {'application': 'firefox',
-              'build_number': '1',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1-build1.en-US.win32.exe',
-     'url': 'firefox/candidates/23.0.1-candidates/build1/win32/en-US/Firefox Setup 23.0.1.exe'},
+@pytest.mark.parametrize("args,filename,url", [
+    ({'application': 'firefox', 'platform': 'linux', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.en-US.linux.tar.bz2',
+     'firefox/candidates/23.0.1-candidates/build3/linux-i686/en-US/firefox-23.0.1.tar.bz2'),
+    ({'application': 'firefox', 'platform': 'linux64', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.en-US.linux64.tar.bz2',
+     'firefox/candidates/23.0.1-candidates/build3/linux-x86_64/en-US/firefox-23.0.1.tar.bz2'),
+    ({'application': 'firefox', 'platform': 'mac', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.en-US.mac.dmg',
+     'firefox/candidates/23.0.1-candidates/build3/mac/en-US/Firefox 23.0.1.dmg'),
+    ({'application': 'firefox', 'platform': 'win32', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.en-US.win32.exe',
+     'firefox/candidates/23.0.1-candidates/build3/win32/en-US/Firefox Setup 23.0.1.exe'),
+    ({'application': 'firefox', 'platform': 'win64', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.en-US.win64.exe',
+     'firefox/candidates/23.0.1-candidates/build3/win64/en-US/Firefox Setup 23.0.1.exe'),
+    ({'application': 'firefox', 'locale': 'de', 'platform': 'win32', 'version': '23.0.1'},
+     'firefox-23.0.1-build3.de.win32.exe',
+     'firefox/candidates/23.0.1-candidates/build3/win32/de/Firefox Setup 23.0.1.exe'),
+    ({'application': 'firefox', 'build_number': '1', 'platform': 'win32', 'version': '23.0.1'},
+     'firefox-23.0.1-build1.en-US.win32.exe',
+     'firefox/candidates/23.0.1-candidates/build1/win32/en-US/Firefox Setup 23.0.1.exe'),
     # -a firefox -p win32 -v 21.0 --stub (old format)
```

Comment:
A simple `# stub-installer (old format)` would be better. We don't need all the other information.